### PR TITLE
Create ScheduledTaskCreationUpdateFromUserWritableDrectory.yaml

### PR DESCRIPTION
**Pull request by vpaschalidis**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Review by YaronFruchtmann — COMMENTED** [review]
I commented over one file, but probably can apply to all

File: Hunting Queries/SecurityEvent/ScheduledTaskCreationUpdateFromUserWritableDrectory.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+tactics:
+  - Execution
+relevantTechniques:
+  - T1053
+query: |
+   // Enter a reference list for writable user paths"
+   let WritableUserPaths = dynamic (["\"C:\\Users\\","\"C:\\ProgramData\\"]);
+   SecurityEvent
+   | where EventID in (4698,4702)
+   | parse EventData with * 'Command&gt;' Command "&lt" *
```
[the remark this answers is not in the document]
Comment (vpaschalidis) [reply Hunting Queries/SecurityEvent/ScheduledTaskCreationUpdateFromUserWritableDrectory.yaml]: Yes, it is &gt.

File: Hunting Queries/SecurityEvent/ScheduledTaskCreationUpdateFromUserWritableDrectory.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+   | parse EventData with * 'Arguments&gt;' Arguments "&lt" *
+   | where Command has_any (WritableUserPaths) or Arguments has_any (WritableUserPaths)
+   | parse EventData with * 'SubjectUserName">' SubjectUserName "<" *
+   | parse EventData with * 'SubjectDomainName">' SubjectDomainName "<" *
+   | parse EventData with * 'TaskName">' TaskName "<" *
```
[the remark this answers is not in the document]
Comment (vpaschalidis) [reply Hunting Queries/SecurityEvent/ScheduledTaskCreationUpdateFromUserWritableDrectory.yaml]: Fixed

File: Hunting Queries/SecurityEvent/ScheduledTaskCreationUpdateFromUserWritableDrectory.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+relevantTechniques:
+  - T1053
+query: |
+   // Enter a reference list for writable user paths"
+   let WritableUserPaths = dynamic (["\"C:\\Users\\","\"C:\\ProgramData\\"]);
+   SecurityEvent
+   | where EventID in (4698,4702)
+   | parse EventData with * 'Command&gt;' Command "&lt" *
+   | parse EventData with * 'Arguments&gt;' Arguments "&lt" *
+   | where Command has_any (WritableUserPaths) or Arguments has_any (WritableUserPaths)
```
[the remark this answers is not in the document]
Comment (vpaschalidis) [reply Hunting Queries/SecurityEvent/ScheduledTaskCreationUpdateFromUserWritableDrectory.yaml]: Done

**Review by aprakash13 — APPROVED** [review]
Thanks @vpaschalidis

**Pull request merged**